package-lock.json

```diff
@@ -1,9595 +1,8 @@
 {
   "name": "homebrewery",
   "version": "3.0.4",
-  "lockfileVersion": 2,
+  "lockfileVersion": 1,
```

Comment:
Do you have an old version of NPM? I think since v7 lockfileVersion defaults to 2 when it generates package-lock.json. Just a different format for the same thing I think but make sure you're updated in case it creates accidental conflicts somewhere.